### ISSUE TITLE
SOFTWARE DEVELOPER

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#2455]
SOFTWARE DEVELOPER<br>
http://ift.tt/2pVvqIE<br><p><img alt="Logo.gif?ixlib=rails-2.1" src="http://ift.tt/2qqwRCM"></p>
<p><strong>Headquarters:</strong> Stuttgart, Germany<br><strong>URL:</strong> <a href="http://virtualq.io/">http://virtualq.io/</a></p>
<div><br></div>
<div><span><i>- you can work remote (CET +/-2h) or in our German office in Stuttgart, Germany -</i></span></div>
<div><br></div>
<div><b>About us</b></div>
<div><br></div>
<div><span>virtualQ provides a&nbsp;<b>novel cloud-solution</b>&nbsp;that waits on hold during longer waiting times for callers of Customer Service Center. Callers can spend their time productively until virtualQ alerts them to call back when it is their turn. Thereby, the&nbsp;<b>virtualQ enterprise software</b>&nbsp;provides an&nbsp;<b>unique customer experience</b>, which is&nbsp;<b>revolutionising the service branche</b>. Well-known companies like&nbsp;<b>Albatros Lufthansa</b>, the German insurances&nbsp;<b>HanseMerkur</b>&nbsp;and&nbsp;<b>Signal Iduna</b>&nbsp;and the international cashback and shopping point provider&nbsp;<b>Lyoness</b>&nbsp;are already using our software. Several national and international institutions awarded the innovative cloud-products of virtualQ and the company is backed by an institutional investment fund. We are currently located in Stuttgart and Berlin, Germany.</span></div>
<div><br></div>
<div>
<span>You get the chance to work on an innovative and awarded product with&nbsp;</span><b>top-notch technology and software management tools</b><span>. We have a German-American background and like the concept of&nbsp;</span><b>simplicity, design, flat hierarchy and employee participation</b><span>&nbsp;through options and shares. You have the chance to work in an awesome team atmosphere with lots for individual liberties. Do you want to change the way in which people wait on hold in customer service telephone lines with us?</span>
</div>
<div>Let us know!</div>
<div><br></div>
<div><span><b>With these activities you will have a great impact on our business</b></span></div>
<div><br></div>
<ul>
<li><span>Work closely with our CTO and shape our product</span></li>
<li><span>Develop exciting&nbsp;<b>Backend and Front Applications</b>, using the latest technologies</span></li>
<li><span>Develop&nbsp;<b>A.I. technologies</b>, that will shape the future of virtualQ</span></li>
<li>
<span>Dive into the stimulating world of information- and<br></span> communication technologies and take responsibilty for<br>
different product features</li>
<li>
<span><span>Be creative and work independently<br></span></span><br>
</li>
</ul><div><span><b>With these skills you are our perfect candidate</b></span></div>
<div><br></div>
<ul>
<li><span>You can call yourself a&nbsp;<b>Full Stack Developer</b>? Awesome!</span></li>
<li>
<b><strong>React.js, Amazon AWS</strong></b><span>&nbsp;&ndash; you know the modern way of developing</span>
</li>
<li><span>Frameworks like&nbsp;<b>MEAN Stack, Ruby/Rails and Node.js</b>&nbsp;feel like home to you</span></li>
<li><span>Of course, you also know how to communicate in an effective manner</span></li>
<li><span>You speak German OR English fluently (native or 2nd language)</span></li>
<li><span>You love working in an international environment</span></li>
<li><span>You do not take things for granted, instead you question the status quo</span></li>
<li><span>You have got 2 &ndash; 5 years of work experience</span></li>
<li>
<span>You have worked in the Telecommunication market before? Or are you interested in learning about it? Awesome!<br></span><b><br></b>
</li>
</ul><div><span><b>Sounds like the perfect job? We have got even more to offer</b></span></div>
<div><br></div>
<ul>
<li>
<span>Your chance to get to know us: You start with a 6 month contract, if we are a good<br></span> fit, there are great chances for a permanent hire!</li>
<li><span>A fair salary &ndash; we might be a startup, but we pay like a company</span></li>
<li><span>Employee participation through options and shares</span></li>
<li><span>Flexibility at its best: work remotely from home or join us at our German office in Stuttgart</span></li>
<li><span>Results Only Work Environment &ndash; we do not care when and where you work, we care about your work results. Therefore, we do not count working hours and holidays either</span></li>
<li><span>Plenty of freedom and opportunities to grow in a young company, which will turn the Service Center market upside down</span></li>
<li><span>Workshops? Seminars? BarCamps? Tell us what you need to develop your skills and we will make it happen</span></li>
<li><span>An awesome, international, open minded team</span></li>
</ul><div><br></div>
<div>Download JobAd (PDF):&nbsp;http://bit.ly/software_developer_virtualQ</div>
<div>&nbsp;</div>
<div><br></div>
<p><strong>To apply:</strong> Anika is looking forward to receiving your application: <a href="mailto:%44%72%65%61%6d%4a%6f%62@%76%69%72%74%75%61%6c%51.%69%6f">DreamJob@virtualQ.io</a></p>
<br><br>
via Remote Jobr Programming Jobs http://ift.tt/1WipnMk<br>
May 17, 2017 at 03:05AM
